docs/networkpolicy-recommendation.md

```diff
@@ -0,0 +1,143 @@
+# NetworkPolicy Recommendation
+
+## Table of Contents
+
+<!-- toc -->
+- [Introduction](#introduction)
+- [Prerequisite](#prerequisite)
+- [Perform NetworkPolicy Recommendation](#perform-networkpolicy-recommendation)
+  - [Run a policy recommendation job](#run-a-policy-recommendation-job)
+  - [Check the status of a policy recommendation job](#check-the-status-of-a-policy-recommendation-job)
+  - [Retrieve the result of a policy recommendation job](#retrieve-the-result-of-a-policy-recommendation-job)
+<!-- /toc -->
+
+## Introduction
+
+Theia NetworkPolicy Recommendation recommends the NetworkPolicy configuration
+to secure Kubernetes network and applications. It analyzes the network flows
+collected by [Grafana Flow Collector](
+network-flow-visibility.md#grafana-flow-collector) to generate
+[Kubernetes NetworkPolicies](
+https://kubernetes.io/docs/concepts/services-networking/network-policies/)
+or [Antrea NetworkPolicies](
+https://github.com/antrea-io/antrea/blob/main/docs/antrea-network-policy.md).
+This feature assists cluster administrators and app developers in securing
+their applications according to Zero Trust principles.
+
+## Prerequisite
+
+- [Flow Exporter](
```

Comment:
Could we simply point to network-flow-visibility.md which should include a section to install everything?

Reply:
Yes, we will change this link to a quick-start section in `network-flow-visibility.md` in the future. Once we finalized the installation story, adding the helm chart for Flow Aggregator, etc.

Reply:
Could we have the change in Theia 0.1? Again, the current installation steps are too hard for users to follow.

Reply:
Got it, according to the discussion inside team yesterday, we plan to add a Flow Aggregator chart in Theia first since Antrea won't ship helm charts in v1.7, and the whole installation steps will be `kubectl apply -f antrea.yaml && helm install flow-aggregator && helm install theia`. 
For the work estimate of this change, I would like to transfer this question to @salv-orlando 
cc. @yanjunz97

Reply:
Even if we cannot get charts for Flow Aggregator in 1.7, let us at least clean up installation section in network-flow-visibility.md to document the basic steps to stet up everything with default settings, and then this doc can point to that.

Reply:
We plan to have a PR containing the Flow Aggregator chart and quick-start section merged by the end of this week.